pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarSchemaHandlers.java

```diff
@@ -50,4 +51,11 @@ static SchemaHandler newPulsarSchemaHandler(SchemaInfo schemaInfo,
         }
     }
 
+    static SchemaInfo defaultSchema(String name) {
+        return SchemaInfo.builder()
```

Comment:
you can simplify with `Schema.BYTES.getSchemaInfo()`. This also avoids creating a schema info object again and again.

Reply:
fix it.